packages/tabs/src/Tabs.spec.tsx

```diff
@@ -117,6 +116,7 @@ describe('packages/Tabs', () => {
       setTimeout(() => {
         const firstContent = getByText('First Content');
         expect(firstContent).toBeInTheDOM();
+        expect(container.querySelectorAll('[role="tabpanel"]').length).toBe(1);
```

Comment:
Are these assertions being called? How does jest know to wait for the setTimeout?

Reply:
You're right, removed!